azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -369,6 +371,35 @@ def __cleanup_unit_file(path):
                 except Exception as exception:
                     _log_cgroup_warning("Failed to remove {0}: {1}", path, ustr(exception))
 
+        @staticmethod
+        def __cleanup_directory_tree(path):
```

Comment:
we should probably delete only the files we create, since somebody else (e.g. VM admin) may have created additional files

Reply:
addressed